pydantic_ai_slim/pydantic_ai/durable_exec/temporal/_agent.py

```diff
@@ -82,6 +86,14 @@ def __init__(
         Args:
             wrapped: The agent to wrap.
             name: Optional unique agent name to use in the Temporal activities' names. If not provided, the agent's `name` will be used.
+            additional_models:
+                Optional mapping or sequence of additional models to register with the agent.
+                - When providing a mapping, keys define the names that can be referenced at runtime and the values are either `Model` instances or model strings (e.g., `'openai:gpt-4'`).
+                - When providing a sequence, only provider/model strings are allowed and they will be referenced by their literal string when calling `run(model=...)`.
+                Model instances must be registered via the mapping form so they can be referenced by name.
+            provider_factory:
+                Optional callable used when instantiating models from provider strings (both pre-registered and those supplied at runtime).
```

Comment:
Is this also used on the model name the agent is initially created with? I would kind of expect it to